src/smart-components/request/request-detail/action-transcript.js

```diff
@@ -6,15 +6,15 @@ import {
   Text,
   TextVariants
 } from '@patternfly/react-core';
-import { CheckIcon, CommentIcon, EnvelopeIcon, OutlinedTimesCircleIcon, AngleDoubleRightIcon } from '@patternfly/react-icons';
+import { CheckCircleIcon, CommentIcon, EnvelopeIcon, OutlinedTimesCircleIcon, AngleDoubleRightIcon } from '@patternfly/react-icons';
 
 const operationDisplayName = { memo: 'Comment from', approve: 'Approved By',
   deny: 'Denied by', notify: 'Notified By', skip: 'Skipped by' };
 
-const operationIcon = { memo: <CommentIcon/>, approve: <CheckIcon/>,
+const operationIcon = { memo: <CommentIcon/>, approve: <CheckCircleIcon fill="#32CD32"/>,
```

Comment:
Just a small comment here. I think it the green color should be `#92D400`. You can see some color guide lines for PF here: https://www.patternfly.org/v4/design-guidelines/styles/colors

Reply:
You are right - I replaced this with the pf4 success color, and added the red one for 'denied'